_posts/2019-02-04-putting-a-plug-app-on-heroku.md

```diff
@@ -0,0 +1,98 @@
+---
+author: Bobby Grayson
+categories: general
+date: 2019-02-03
+layout: post
+title: Getting a Plug App Working With Heroku
+---
+
+In our [previous_post](TODO_LINK) we built a very simple Portfolio app with `Plut.Router`.
+In this (brief) post, we will show how to get things up and running on [Heroku](http://heroku.com) for a vanilla `Plug` app.
+It really is quite easy, but today when I set out to do it, the resources I found either focused on Phoenix or had a few holes, so I figured I would write up a simple one here.
+
+## Let's Do It!
+To deploy to Heroku, your first step is always going to be your `Procfile`.
+The `Procfile` essentially determines what boots up your app.
+Ours is very simple, we will put it at the top level.
+
+```
+web: mix run --no-halt
+```
+
+Thats it!
+
+Now, the reason Heroku is so easy is because they have a _lot_ of built in tooling to make deploying your app easy.
+In the case of Elixir, we need a buildpack that will handle our compilation and to run the app and get the proper dependencies on our server.
+To get any further, you will need a Heroku account and to have their toolbelt installed.
```

Comment:
I think we should cover setting up your Heroku account earlier and almost as a footnote, it's not the focus of this post _really_.